src/common/helpers/helpers.ts

```diff
@@ -171,6 +171,24 @@ export function getBoardId(): number {
     return id;
 }
 
+export function getBoardPublicLink(): string {
+    const publicLink = window["FollozeState"]?.initialState?.board?.public_link;
+    // available only in designer
```

Comment:
seems redundant

Reply:
I'm not sure I agree. You and I didn't know that when I was trying to figure out why there's a fallback to begin with.
I think I'd have an easier time working with this function if I knew in which cases this value wasn't available.

Reply:
Found a good compromise: there was already an "Only in designer" comment in IBoard.ts